instrumentation/helpers-mysql/lib/opentelemetry/instrumentation/helpers/mysql.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0module OpenTelemetry
+
+module OpenTelemetry
+  module Instrumentation
+    module Helpers
+      # Helpers for MySQL instrumentation libraries, like mysql2 and trilogy
+      class MySQL
```

Comment:
@ericmustin  - Is `instrumentation` the correct parent folder for these helper gems? Since propagators and instrumentation are on a peer level, I'm wondering if we should add something like that for the helper gems too. However, these helpers seem like they'd only be used by instrumentation gems, so maybe this makes sense. It's a minor thing, but I want to make sure I'm establishing a good pattern.

Reply:
yea it's a good question i think the pattern in https://github.com/open-telemetry/opentelemetry-ruby is for these to live at the peer level, ala `common`, but idk if we want to have like a `common/helpers` folder with both these in it or something like that. So tbh i'm kinda fine with `instrumentation` for now but mostly because I think someone else will have a stronger opinion.

Reply:
slack thread https://cloud-native.slack.com/archives/C01NWKKMKMY/p1688738760694089?thread_ts=1688672784.020679&cid=C01NWKKMKMY

Reply:
We ultimately decided to move these into a helpers directory, outside of instrumentation. The names for the new gems are `opentelemetry-helpers-<type>`.